### ISSUE TITLE
Flist request: XRDP

### ISSUE THREAD

**Issue by mik-tf** [threefoldtech/tf-images#304]
# Situation

A user wants an xrdp flist. On full vm preferably as I understand.

"Yes, I usually use this tutorial. However, I have over 40 VMs and I hope to have a ready-made VM flist instead of requiring each VM to install its own desktop.  https://manual.grid.tf/documentation/system_administrators/getstarted/remote-desktop_gui/xrdp_guide/xrdp_guide.html" link https://t.me/threefold/1/61687

# Todo

- create xrdp ubuntu flist with non root user

# Notes

- I understand if we decide to not support this as an official image.

**Comment by mik-tf** [comment]
# Update

- All the work has been done in this repo: https://github.com/Mik-TF/ubuntu_xrdp_fullvm_flist
  - This includes the procedures and documentation (procs and docs!) 
- The user could deploy it himself and it all worked 100%
- It's now up to ThreeFold to decide if we want to support this as an official image

@PeterNashaat @coesensbert  @hossnys 
Dear codeowners, wdyt?

If you think it's a good idea, I can make a PR to this repo.

**Comment by hossnys** [comment]
Thanks Mik for that effort , but do you mean with that issue?
1- to have this as an official image only 
 or 
2- to have a solution for it in dashboard that user can deploy it from there 
if your answer is number 1 so you can create a PR for your changes , but if it's number 2 so it needs to 
be discussed with dev team @xmonader @AhmedHanafy725

**Comment by mik-tf** [comment]
@hossnys I think we can go with 1 for now. The user seemed happy with using it via the micro VM page.

If the dev team thinks it would be a nice addition to the Dashboard, I am all for it though. But it might be sufficient as 1. for now.

**Comment by mik-tf** [comment]
Also, this Flist is to help a user with a specific use case and to showcase the possibilities. IMO it should be documented but it doesn't have to become an official images.

We can consider having a category of flists that are in this repo but are not officially supported, i.e. we have the official images that we keep maintaining and present as official, and we can have other flists + variations on official images, such as this Ubuntu 24.04 XRDP flist.

**Comment by hossnys** [comment]
@mik-tf , can we close this one as the [PR](https://github.com/threefoldtech/tf-images/pull/307) related to it is already merged , then you can discuss with dev team about having it documented only , or as official one .

**Comment by mik-tf** [comment]
@hossnys  I agree. Let's close it. We won't need it as an official image. It was OK to show it is possible and documented. Thanks.
